notebooks/MAST/plot_analyze_with_lightkurve/plot_analyze_with_lightkurve.ipynb

```diff
@@ -0,0 +1,953 @@
+{
```

Comment:
<p>Let's discuss how to install extra packages in #11</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/ScienceBetter/notebooks-stsci/pull/13/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>